src/Components/Molecules/FMSearch/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { memo } from "react";
+import {
+  FormControl,
+  TextField,
+  InputAdornment,
+  FormHelperText,
+} from "@mui/material";
+import { IFMSearch } from "./FMSearch.interfaces";
+
+export const FMSearch = memo((props: IFMSearch) => {
+  const {
+    disabled,
+    variants,
+    type,
+    testID,
+    value,
+    label,
+    helperText,
+    onChange,
+    ...materialUIProps
+  } = props;
+
+  return (
+    <FormControl data-testid={testID}>
+      <TextField
```

Comment:
I think we just need to create <FormControl> and its helper text in molecules component and a child form atom component (eg: TextField, Select, RadioButton, CheckBox, etc).

Reply:
nicely done